src/towncrier/test/test_build.py

```diff
@@ -568,55 +607,79 @@ def test_singlefile_errors_and_explains_cleanly(self):
             result.output,
         )
 
-    def test_single_file_false(self):
+    def test_single_file(self):
```

Comment:
Maybe a better description for the feature

```suggestion
    def test_all_version_notes_in_a_single_file(self):
```